.pylintrc

```diff
@@ -16,14 +16,15 @@ persistent=yes
 # List of plugins (as comma separated values of python modules names) to load,
 # usually to register additional checkers.
 load-plugins=
+disable=multiple_types
 
 
 [MESSAGES CONTROL]
 
 # Enable the message, report, category or checker with the given id(s). You can
 # either give multiple identifier separated by comma (,) or put this option
 # multiple time.
-#enable=
+enable=C
```

Comment:
Why?

Reply:
Oh, this was an attempt to get notification for the bad indents of continuation lines, but didn't work... I can remove it!

Reply:
Removed